dashboards-observability/common/utils/query_utils.ts

```diff
@@ -59,7 +59,7 @@ export const buildQuery = (baseQuery: string, currQuery: string) => {
   if (baseQuery) {
     fullQuery = baseQuery;
     if (currQuery) {
-      fullQuery += '| ' + currQuery;
+      fullQuery += currQuery.trim().charAt(0) !== '|' ? '| ' : '' + currQuery;
```

Comment:
The query that comes after the base query in app analytics should not start with a | and the autocomplete should make sure of this by suggesting the commands. Are we adding this just in case the user ignores the autocomplete and adds a pipe? This works but am just curious why it's necessary?

Reply:
Also, parentheses need to be added around `currQuery.trim().charAt(0) !== '|' ? '| ' : ''`, or else it will be processed as currQuery.trim().charAt(0) !== '|' ? `('| ')` : `('' + currQuery)` which misses the currQuery after the pipe.

Reply:
We have eliminated the above implementation